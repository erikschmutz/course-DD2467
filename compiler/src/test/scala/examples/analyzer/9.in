let f a b = a b
let add c = c + 1
let res = f add 2