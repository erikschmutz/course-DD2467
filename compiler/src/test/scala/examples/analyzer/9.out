AnalyzerResult(
    Integer(),
    Enviroment(
        List(
            ValEntry(
                Identifier(g),
                Integer()
            ),
            ValEntry(
                Identifier(f),
                Function(Unknown('a),Integer())
            )
        )
    )
)